packages/eas-cli/src/commands/submit.ts

```diff
@@ -33,6 +33,7 @@ interface RawCommandFlags {
   verbose: boolean;
```

Comment:
We already have the `verbose` flag. Maybe instead of adding another flag, we can alter the behavior of `eas submit` when `--verbose` is specified